docs/release-notes/1.0.0.md

```diff
@@ -1,3 +1,6 @@
+After much anticipation, numerous test releases, and an immense amount of hard work, we proudly announce the first production release 1.0.0 of the Istio module. This module will now be added to the collection of other modules that have already been separated from the classic Kyma as part of the [modularization process](https://github.com/kyma-project/istio/blob/main/docs/contributor/01-00-installation.md). 
```

Comment:
I'm not sure if you mean to link to the installation here. Did you mean maybe the document about [Kyma modules](https://kyma-project.github.io/kyma/#/?id=kyma-modules) or [Modularization](https://github.com/kyma-project/community/tree/aaa980e3bedbf7aa1fd543221baf3e28c3dd4101/concepts/modularization) in `/community`? Or some other document?

Reply:
I didn't! Thanks for spotting that :) I must've copied the same link two times 
I wanted to link to the doc about [Kyma modules ](https://kyma-project.io/docs/kyma/latest/01-overview/#kyma-modules) on `kyma-project.io`, but now I wonder if I should link to this website or to the new one. What do you think?

Reply:
Hmm, the problem is that the one I provided is not the final one (the final one will be https://kyma-project.io/#/). For now, I would link to the old website as you wrote in your comment.